invokeai/backend/model_management/model_probe.py

```diff
@@ -404,19 +404,20 @@ class ControlNetCheckpointProbe(CheckpointProbeBase):
     def get_base_type(self) -> BaseModelType:
         checkpoint = self.checkpoint
         for key_name in (
-            "control_model.input_blocks.2.1.transformer_blocks.0.attn2.to_k.weight",
-            "input_blocks.2.1.transformer_blocks.0.attn2.to_k.weight",
+                "control_model.input_blocks.2.1.transformer_blocks.0.attn2.to_k.weight",
+                "input_blocks.2.1.transformer_blocks.0.attn2.to_k.weight",
+                "down_blocks.1.attentions.0.transformer_blocks.0.attn2.to_k.weight",
         ):
             if key_name not in checkpoint:
                 continue
-            if checkpoint[key_name].shape[-1] == 768:
+            width = checkpoint[key_name].shape[-1]
+            if width == 768:
                 return BaseModelType.StableDiffusion1
-            elif checkpoint[key_name].shape[-1] == 1024:
+            elif width == 1024:
                 return BaseModelType.StableDiffusion2
-            elif self.checkpoint_path and self.helper:
-                return self.helper(self.checkpoint_path)
-        raise InvalidModelException("Unable to determine base type for {self.checkpoint_path}")
-
+            elif width == 2048:
+                return BaseModelType.StableDiffusionXL
+        raise InvalidModelException("{self.model_path}: Unable to determine base type")
```

Comment:
This should be an f-string, and `self.model_path` does not exist:
```suggestion
        raise InvalidModelException(f"{self.checkpoint_path}: Unable to determine base type")
```